src/main/java/gov/cabinetoffice/gap/adminbackend/services/GrantAdvertService.java

```diff
@@ -416,8 +415,7 @@ private void addFieldToContentfulAdvert(final CMAEntry contentfulAdvert,
         contentfulAdvert.setField(questionResponse.getId(), CONTENTFUL_LOCALE, contentfulValue);
     }
 
-    // TODO ideally this is just a stop gap until we can use the CMA library to convert
-    // the rich text strings to CMARichDocument objects
+    // TODO ideally this is just a stop gap until we can use the CMA library to convert the rich text strings to CMARichDocument objects
```

Comment:
Wondering if we can just remove this comment as it's not something we are going to do

Reply:
yeah had the same thought yesterday haha!